style.css

```diff
@@ -0,0 +1,103 @@
+
+* {
```

Comment:
very general style, removes all margins and paddings from all elements, I would recommend to avoid using such selectors